rpa-recorder/6.4/modules/ROOT/pages/user-interface.adoc

```diff
@@ -27,6 +28,46 @@ The *Recording Toolbar* appears on the screen when RPA Recorder starts recording
 
 When a recording state changes, a toast notification appears indicating current state. The notification stays for two seconds and disappears automatically.
 
+== Image-Based Mouse Clicks
+
+Use image-based mouse clicks at any point during a normal recording session to record mouse clicks on a UI element that otherwise cannot be recorded properly. This feature guides you through the mouse click process. An image on the screen and mouse clicks must be captured, and the mouse clicks are recorded relative to the captured image.
```

Comment:
suggestion: remove the word "normal" here and below. I guess that it was from our PM, and the word is commonly used in German. There is just one "recording session" (neither normal nor abnormal) with two possible activity types - "Bot Task" or "Web Task". IBMC is available in both, just as "Add screenshot" is available in both. That's what this sentence is trying to say.
suggestion: how about changing "_This feature guides you through the mouse click process_." to "_This feature guides you through the capture of image and mouse clicks._ " 
praise: the rest is good and clear!